---
layout: page
title: Publications
permalink: /publications/
---

# JOURNAL ARTICLES

#### Oral microbial signatures associated with age and frailty in Canadian adults.
[Link to paper](https://doi.org/10.1038/s41598-024-60409-8) / [Link to preprint](https://www.researchsquare.com/article/rs-3760138/v1)<br>
DeClercq, V., **Wright, R.**, Nearing, J. & Langille, M. (2024) *Sci. Rep.* <br>
**Contribution:** data analysis, writing and editing of the manuscript.<br>
<details>
  <summary>Abstract</summary>
  This study aimed to assess the association between the oral microbiome, age, and frailty. Data and saliva samples were obtained from male and female participants aged 35–70 years (n = 1357). Saliva samples were analysed by 16S rRNA gene sequencing and differences in microbial diversity and community compositions were examined in relation to chronological age and the frailty index (FI). Most alpha diversity measures (Richness, Shannon Diversity, Faith’s Phylogenetic Diversity) showed an inverse association with frailty, whereas a positive association was observed with age and Shannon Diversity and Evenness. A further sex-stratified analysis revealed differences in measures of microbial diversity and composition. Multiple genera were detected as significantly differentially abundant with increasing frailty and age by at least two methods. With age, the relative abundance of Veillonella was reduced in both males and females, whereas increases in Corynebacterium appeared specific to males and Aggregatibacter, Fusobacterium, Neisseria, Stomatobaculum, and Porphyromonas specific to females. Beta diversity was significantly associated with multiple mental health components of the FI. This study shows age and frailty are differentially associated with measures of microbial diversity and composition, suggesting the oral microbiome may be a useful indicator of increased risk of frailty or a potential target for improving health in ageing adults.
</details>
<br>
#### Microbial hitchhikers harbouring antimicrobial-resistance genes in the riverine plastisphere.
[Link to paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01662-3) / [Link to preprint](https://www.researchsquare.com/article/rs-2886255/v1)<br>
Zadjelovic, V., **Wright, R.**, Borsetto, C., Jeannelle Quartey, Tyler N. Cairns, Langille, M., Quilliam, R., Christie-Oleza, J. & Wellington, E. (2023)  *Microbiome.*<br>
**Contribution:** all data analysis and figure generation; writing and editing of the manuscript.<br>
<details>
<summary>Abstract</summary>
<b>Background</b><br>
The widespread nature of plastic pollution has given rise to wide scientific and social concern regarding the capacity of these materials to serve as vectors for pathogenic bacteria and reservoirs for Antimicrobial Resistance Genes (ARG). In- and ex-situ incubations were used to characterise the riverine plastisphere taxonomically and functionally in order to determine whether antibiotics within the water influenced the ARG profiles in these microbiomes and how these compared to those on natural surfaces such as wood and their planktonic counterparts.<br>
<br>
<b>Results</b><br>
We show that plastics support a taxonomically distinct microbiome containing potential pathogens and ARGs. While the plastisphere was similar to those biofilms that grew on wood, they were distinct from the surrounding water microbiome. Hence, whilst potential opportunistic pathogens (i.e. Pseudomonas aeruginosa, Acinetobacter and Aeromonas) and ARG subtypes (i.e. those that confer resistance to macrolides/lincosamides, rifamycin, sulfonamides, disinfecting agents and glycopeptides) were predominant in all surface-related microbiomes, especially on weathered plastics, a completely different set of potential pathogens (i.e. Escherichia, Salmonella, Klebsiella and Streptococcus) and ARGs (i.e. aminoglycosides, tetracycline, aminocoumarin, fluoroquinolones, nitroimidazole, oxazolidinone and fosfomycin) dominated in the planktonic compartment. Our genome-centric analysis allowed the assembly of 215 Metagenome Assembled Genomes (MAGs), linking ARGs and other virulence-related genes to their host. Interestingly, a MAG belonging to Escherichia –that clearly predominated in water– harboured more ARGs and virulence factors than any other MAG, emphasising the potential virulent nature of these pathogenic-related groups. Finally, ex-situ incubations using environmentally-relevant concentrations of antibiotics increased the prevalence of their corresponding ARGs, but different riverine compartments –including plastispheres– were affected differently by each antibiotic.<br>
<br>
<b>Conclusions</b><br>
Our results provide insights into the capacity of the riverine plastisphere to harbour a distinct set of potentially pathogenic bacteria and function as a reservoir of ARGs. The environmental impact that plastics pose if they act as a reservoir for either pathogenic bacteria or ARGs is aggravated by the persistence of plastics in the environment due to their recalcitrance and buoyancy. Nevertheless, the high similarities with microbiomes growing on natural co-occurring materials and even more worrisome microbiome observed in the surrounding water highlights the urgent need to integrate the analysis of all environmental compartments when assessing risks and exposure to pathogens and ARGs in anthropogenically-impacted ecosystems.<br>
</details>
<br>
#### Assessing the impact of chronic and acute plastic pollution from construction activities and other anthropogenic sources: A case study from the coast of Antofogasta, Chile.
[Link to paper](https://doi.org/10.1016/j.marpolbul.2023.115510) / [Link to preprint](http://dx.doi.org/10.2139/ssrn.4489230)<br>
Zadjelovic, V., **Wright, R.**, Walker, T., Avalos, V., Marîn, P., Ushak, S., Christie-Oleza, J. & Riquelme, C. (2023) *Mar. Poll. Bull.*<br>
**Contribution:** writing and editing of the manuscript.<br>
<details>
<summary>Abstract</summary>
Plastic pollution is a critical environmental issue with far-reaching and not yet fully explored consequences. This study uncovered a significant source of plastic contamination arising from improper application and management of expanded polystyrene (EPS) utilised as expansion joints at a construction site near the coast of Antofagasta, Chile. Through meticulous field observations and calculations, we estimate that a staggering 82.9 million EPS spheres have the potential to be released into the environment from the 7.62 m3 of this material used for the construction of this coastal promenade, constituting a chronic source of pollution. Despite the ongoing construction, we have already evidenced mechanical fragmentation and dispersion of EPS microplastic pollution in the surrounding natural environment. To our knowledge, this is the first study that documents misused construction materials contributing to plastic pollution. In addition to the EPS pollution, our findings reveal an alarming accumulation of litter – an acute pollution source – including plastic cups, bottles, carrier bags, and several other construction materials (e.g. plastic nets, films) that are exacerbating the pollution problems within the region and potentially endangering marine and terrestrial organisms. These observations highlight the urgent need for mitigating measures and intervention policies targeting construction-related plastic and microplastic pollution, along with a more robust regulatory framework for construction activities as well as adequate surveillance and enforcement.
</details>
<br>
#### Exploring the microbiome of oral epithelial dysplasia as a predictor of malignant progression. 
[Link to paper](https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-023-02911-5) / [Link to preprint](https://www.researchsquare.com/article/rs-2129013/v1)<br>
**Wright, R.**, Pewarchuk, M., Marshall, E., Murray, B., Rosin, M., Laronde, D., Zhang, L., Lam, W., Langille, M. & Rock, L. (2023) *BMC Oral Health.*<br>
<details>
<summary>Abstract</summary>
A growing body of research associates the oral microbiome and oral cancer. Well-characterized clinical samples with outcome data are required to establish relevant associations between the microbiota and disease. The objective of this study was to characterize the community variations and the functional implications of the microbiome in low-grade oral epithelial dysplasia (OED) using 16S rRNA gene sequencing from annotated archival swabs in progressing (P) and non-progressing (NP) OED. We characterised the microbial community in 90 OED samples — 30 swabs from low-grade OED that progressed to cancer (cases) and 60 swabs from low-grade OED that did not progress after a minimum of 5 years of follow up (matched control subjects). There were small but significant differences between P and NP samples in terms of alpha diversity as well as beta diversity in conjunction with other clinical factors such as age and smoking status for both taxa and functional predictions. Across all samples, the most abundant genus was Streptococcus, followed by Haemophilus, Rothia, and Neisseria. Taxa and predicted functions were identified that were significantly differentially abundant with progression status (all Ps and NPs), when samples were grouped broadly by the number of years between sampling and progression or in specific time to progression for Ps only. However, these differentially abundant features were typically present only at low abundances. For example, Campylobacter was present in slightly higher abundance in Ps (1.72%) than NPs (1.41%) and this difference was significant when Ps were grouped by time to progression. Furthermore, several of the significantly differentially abundant functions were linked to the Campylobacteraceae family in Ps and may justify further investigation. Larger cohort studies to further explore the microbiome as a potential biomarker of risk in OED are warranted.
</details>
<br>
#### From defaults to databases: parameter and database choice dramatically impact the performance of metagenomic taxonomic classification tools.
**Wright, R.**, Comeau, A. & Langille, M. (2023) *Microb. Genom.* ([link to paper](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000949) / [link to preprint](https://www.biorxiv.org/content/10.1101/2022.04.27.489753v1))<br>
<details>
<summary>Abstract</summary>
In metagenomic analyses of microbiomes, one of the first steps is usually the taxonomic classification of reads by comparison to a database of previously taxonomically classified genomes. While different studies comparing metagenomic taxonomic classification methods have determined that different tools are ‘best’, there are two tools that have been used the most to-date: Kraken (k-mer-based classification against a user-constructed database) and MetaPhlAn (classification by alignment to clade-specific marker genes), the latest versions of which are Kraken2 and MetaPhlAn 3, respectively. We found large discrepancies in both the proportion of reads that were classified as well as the number of species that were identified when we used both Kraken2 and MetaPhlAn 3 to classify reads within metagenomes from human-associated or environmental datasets. We then investigated which of these tools would give classifications closest to the real composition of metagenomic samples using a range of simulated and mock samples and examined the combined impact of tool–parameter–database choice on the taxonomic classifications given. This revealed that there may not be a one-size-fits-all ‘best’ choice. While Kraken2 can achieve better overall performance, with higher precision, recall and F1 scores, as well as alpha- and beta-diversity measures closer to the known composition than MetaPhlAn 3, the computational resources required for this may be prohibitive for many researchers, and the default database and parameters should not be used. We therefore conclude that the best tool–parameter–database choice for a particular application depends on the scientific question of interest, which performance metric is most important for this question and the limit of available computational resources.
</details>
<br>
#### Microbial pioneers of plastic colonisation in coastal seawaters. 
[Link to paper](https://doi.org/10.1016/j.marpolbul.2022.113701)<br>
Latva, M., Dedman, C., **Wright, R.**, Polin, M. & Christie-Oleza, J. (2022) *Mar. Poll. Bull.*<br>
**Contribution:** discussion of experimental and statistical methodologies, MiSeq library preparation and sequencing, supervision of data analysis, editing of the manuscript.<br>
<details>
<summary>Abstract</summary>
Plastics, when entering the environment, are immediately colonised by microorganisms. This modifies their physico-chemical properties as well as their transport and fate in natural ecosystems, but whom pioneers this colonisation in marine ecosystems? Previous studies have focused on microbial communities that develop on plastics after relatively long incubation periods (i.e., days to months), but very little data is available regarding the earliest stages of colonisation on buoyant plastics in marine waters (i.e., minutes or hours). We conducted a preliminary study where the earliest hours of microbial colonisation on buoyant plastics in marine coastal waters were investigated by field incubations and amplicon sequencing of the prokaryotic and eukaryotic communities. Our results show that members of the Bacteroidetes group pioneer microbial attachment to plastics but, over time, their presence is masked by other groups – Gammaproteobacteria at first and later by Alphaproteobacteria. Interestingly, the eukaryotic community on plastics exposed to sunlight became dominated by phototrophic organisms from the phylum Ochrophyta, diatoms at the start and brown algae towards the end of the three-day incubations. This study defines the pioneering microbial community that colonises plastics immediately when entering coastal marine environments and that may set the seeding Plastisphere of plastics in the oceans.
</details>
<br>
#### Microbiome differential abundance methods produce different results across 38 datasets. 
[Link to paper](https://doi.org/10.1038/s41467-022-28034-z) / [Link to preprint](https://www.biorxiv.org/content/10.1101/2021.05.10.443486v1)<br>
Nearing, J., Douglas, G., Hayes, M., MacDonald, J., Desai, D., Allward, N., Jones, C., **Wright, R.**, Dhanani, A., Comeau, A. & Langille, M. (2022) *Nat. Comms.*<br>
**Contribution:** processed some of the datasets, contributed to decision-making on analyses and presentation of results, editing of the manuscript.<br>
<details>
<summary>Abstract</summary>
Identifying differentially abundant microbes is a common goal of microbiome studies. Multiple methods are used interchangeably for this purpose in the literature. Yet, there are few large-scale studies systematically exploring the appropriateness of using these tools interchangeably, and the scale and significance of the differences between them. Here, we compare the performance of 14 differential abundance testing methods on 38 16S rRNA gene datasets with two sample groups. We test for differences in amplicon sequence variants and operational taxonomic units (ASVs) between these groups. Our findings confirm that these tools identified drastically different numbers and sets of significant ASVs, and that results depend on data pre-processing. For many tools the number of features identified correlate with aspects of the data, such as sample size, sequencing depth, and effect size of community differences. ALDEx2 and ANCOM-II produce the most consistent results across studies and agree best with the intersect of results from different approaches. Nevertheless, we recommend that researchers should use a consensus approach based on multiple differential abundance methods to help ensure robust biological interpretations.
</details>
<br>
### A multi-OMIC study of biodegradation and microbial community succession within the PET Plastisphere. 
**Wright, R.**, Bosch, R., Langille, M., Gibson, M. & Christie-Oleza, J. (2021) *Microbiome.* ([link to paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01054-5) / [link to preprint](https://doi.org/10.21203/rs.3.rs-104576/v1); [link to conference abstract](https://doi.org/10.1099/acmi.ac2019.po0168))
<details>
<summary>Abstract</summary>

</details>
<br>
### Food or just a free ride? A meta-analysis reveals the global diversity of the Plastisphere.
**Wright, R.**, Langille, M. & Walker, T. (2021) *The ISME Journal.* ([link to paper](https://doi.org/10.1038/s41396-020-00814-9))
<details>
<summary>Abstract</summary>

</details>
<br>
### Damage repair versus aging in an individual-based model of biofilms.
**Wright, R.**, Clegg, R., Coker, T. & Kreft, J-U. (2020) *mSystems.* ([link to paper](https://doi.org/10.1128/mSystems.00018-20) / [link to preprint](https://www.biorxiv.org/content/10.1101/2020.01.08.899740v1) / [link to conference abstract](https://www.microbiologyresearch.org/content/journal/acmi/10.1099/acmi.ac2019.po0205))
<details>
<summary>Abstract</summary>

</details>
<br>
### Marine plastic debris – a new surface for microbial colonization. 
**Wright, R.***, Erni-Cassola, G.*, Zadjelovic, V., Latva, M. & Christie-Oleza, J. (2020) *Env. Sci. Technol.* * denotes equal contribution. ([link to paper](https://doi.org/10.1021/acs.est.0c02305) / [link to journal cover](https://pubs.acs.org/toc/esthag/54/19))
<details>
<summary>Abstract</summary>

</details>
<br>
### Plasticizer degradation by marine bacterial isolates: a proteogenomic and metabolomic characterization. 
**Wright, R.**, Bosch, R., Gibson, M. & Christie-Oleza, J. (2020) *Env. Sci. Technol.* ([link to paper](https://pubs.acs.org/doi/10.1021/acs.est.9b05228) / [link to journal cover](https://pubs.acs.org/toc/esthag/54/4))
<details>
<summary>Abstract</summary>

</details>
<br>
### Early colonization of weathered polyethylene by distinct bacteria in marine coastal seawater. 
Erni-Cassola, G., **Wright, R.**, Gibson, M. & Christie-Oleza, J. (2019) *Microb. Ecol.* ([link to paper](https://link.springer.com/article/10.1007/s00248-019-01424-5))
**Contribution:** discussion of experimental and statistical methodologies, MiSeq library preparation and sequencing, editing of the manuscript.
<details>
<summary>Abstract</summary>

</details>
<br>
### Understanding microbial community dynamics to improve optimal microbiome selection.
**Wright, R.**, Gibson, M. & Christie-Oleza, J. (2019) *Microbiome.* ([link to paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0702-x) / [link to preprint](https://doi.org/10.1101/474742))
<details>
<summary>Abstract</summary>

</details>
<br>
# PREPRINTS

### Lime amendment to chronically acidified forest soils results in shifts in microbial communities.
Hosmer, M.*, **Wright, R.***, McCavour, C., Keys, K., Langille, M., Sterling, S. & Rohde, J. * denotes equal contribution. ([link to preprint](https://doi.org/10.21203/rs.3.rs-3876773/v1))
<details>
<summary>Abstract</summary>

</details>
<br>
# BOOK CHAPTERS

### Current research on microbe-plastic interactions in the marine environment. 
*In: Plastic Pollution in the Global Oceans.*
Latva, M., Zadjelovic, V. & **Wright, R.** (2023) Ed. Horton, A. ([link to chapter](https://doi.org/10.1142/9789811259111_0011) / [link to preprint](https://www.preprints.org/manuscript/202107.0273/v1))
**Contribution:** senior author; planned and coordinated the review, carried out reviews of scientific literature and other reviews, analysed data/produced figures, writing and editing of the manuscript.
<details>
<summary>Abstract</summary>

</details>
<br>
### Pollution and Potential Anthropogenic Impacts in Chagos/BIOT.
*In: Coral Reefs of the United Kingdom Overseas Territories.*
Readman, J., […11 other authors in alphabetical order…], **Wright, R.** & Sheppard, C. (2013) Contaminants, ([link to chapter](https://link.springer.com/chapter/10.1007/978-94-007-5965-7_21))
**Contribution:** carried out manual sorting of sediment samples for potential plastic pieces and performed FTIR spectroscopy on all potential plastics for identification.
<details>
<summary>Abstract</summary>

</details>
<br>
# COMMISSIONED SCIENTIFIC ARTICLES

### The global diversity of the plastisphere
**Wright, R.** (2021)  (magazine article for the March 2021 issue of Society for applied Microbiology’s The Microbiologist magazine; [link to magazine](https://sfam.org.uk/knowledge/microbiologist-magazine.html))

### The PET-biodegrading potential of the Plastisphere
**Wright, R.** (2020) (blog post as part of the Microbiology Society’s “A sustainable future” project; [link to article](https://microbiologysociety.org/our-work/75th-anniversary-a-sustainable-future/circular-economy/circular-economy-case-studies/the-pet-biodegrading-potential-of-the-plastisphere.html))

### Marine bacteria and the plastisphere. 
**Wright, R.** (2019) *Biological Sciences Review* (article for educational magazine aimed at 16-18 year-old biology students)

