---
layout: page
title: Resources
permalink: /resources/
---

I have been involved with teaching several workshops as well as maintaining some software/tools and keeping standard operating procedures for the Langille Lab up to date. I am a big proponent for open science and have tried to make all of these as accessible as possible. This page has a non-comprehensive list of all of these.

<h2 style="color:#AF6182;">SOFTWARE/TOOLS/WORKFLOWS</h2>

<h3 style="color:#C57F3B;">Genome Coverage Checker</h3>

Genome Coverage Checker is a tool that I've been developing for checking the coverage of taxa identified by Kraken2 within metagenomic samples. I am currently still working on streamlining and testing this, but it is available [here](https://github.com/R-Wright-1/genome_coverage_checker).

<h3 style="color:#C57F3B;">Kraken2 databases</h3>

Following on from my research comparing the performance of two common metagenomic taxonomic classifiers, Kraken 2 and MetaPhlAn 3 ([From defaults to databases....](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000949)), I have instructions and some helpful scripts on how to build a Kraken 2 database with any genomes of interest [here](https://github.com/R-Wright-1/kraken_metaphlan_comparison/wiki). This Github also contains links to a database that contains all RefSeq V208 genomes. 

<h3 style="color:#C57F3B;">Microbiome Helper</h3>

[Microbiome Helper](https://github.com/LangilleLab/microbiome_helper/wiki) is a Github repository that contains scripts and standard operating procedures for carrying out a range of bioinformatic procedures, such as amplicon sequencing and metagenomic sequencing processing. The first version was published by Gavin Douglas, Andr√© Comeau and Morgan Langille in 2017, and the Github repository has since been worked on by a number of past and present Langille Lab members. I am currently working on streamlining what is there to reflect the methods that we currently use for our analyses, as well as developing a range of tutorials and Amazon Web Services images that may be used by bioinformatics beginners for their own analysis. 

<h3 style="color:#C57F3B;">PICRUSt2</h3>

PICRUSt2 is a bioinformatic tool that allows users to predict microbial community functions based on amplicon sequencing data. Since the end of 2023, I have been assisting PICRUSt2 users with any issues that they may run into that they post about on the [Github page](https://github.com/picrust/picrust2) or on the [picrust-users](https://groups.google.com/g/picrust-users) Google Group. I have also been working on creating a new, updated database (which will be able to be updated more frequently in the future), which is currently in testing by several users. More details on the updated database can be found [here](https://github.com/picrust/picrust2/wiki/Testing-the-updated-PICRUSt2-database). 

<h2 style="color:#AF6182;">WORKSHOPS</h2>

<h3 style="color:#C57F3B;">2024 ICG Microbiome Analysis</h3>

**July 8-17 2024, Halifax (Nova Scotia)**
[Workshop materials](https://github.com/LangilleLab/microbiome_helper/wiki/CBW-2024-workshops)

During this workshop, I mainly replicated what I had taught in the 2024 CBW Microbiome Analysis workshops, but taught these on my own. I also realised that there wasn't enough detail to fully replicate the workshops using the Amazon Instances, so I provided this on the Github page. The page above also includes links to the CBW workshop lectures on YouTube and is where I would recommend starting for anyone looking at trying to do one of the Microbiome Analysis workshops. 

<h3 style="color:#C57F3B;">2024 CBW Microbiome Analysis</h3>

After participating in the teaching of the CBW-IMPACTT workshops in 2022 and 2023, I'd realised that the cohesiveness and redundancy of the different modules (previously taught by different people from different universities) could be improved upon. This workshop followed my new recommendations/curriculum and was split into two parts, Beginner and Advanced Microbiome Analysis. I both lectured, designed lab materials and coordinated the design of lab materials for these workshops. All details are within the workshop materials.

<h4>Beginner Microbiome Analysis</h4>

**May 27-28 2024, St John's (Newfoundland)**
[Workshop materials](https://bioinformaticsdotca.github.io/BMB_2024)

<h4>Advanced Microbiome Analysis</h4>

**May 29-30 2024, St John's (Newfoundland)**
[Workshop materials](https://bioinformaticsdotca.github.io/AMB_2024)

<h3 style="color:#C57F3B;">2023 CBW-IMPACTT Microbiome Analysis</h3>

**July 5-7 2023, Calgary (Alberta)**
[Workshop materials](https://bioinformaticsdotca.github.io/MIC_2023)

I assisted with coordinating the material for Modules 2 ([Marker Gene Taxonomic Analysis (16S)](https://bioinformaticsdotca.github.io/MIC_2023_Module2_lab)) and 3 ([Microbiome Statistics and Visualizations](https://bioinformaticsdotca.github.io/MIC_2023_Module3_lab)) as well as generating all of the lab materials for Module 4 ([Intro to MGS](https://bioinformaticsdotca.github.io/MIC_2023_Module4_lab) including taxonomic and functional annotation of reads). 
